### PR TITLE
Don't stop edge events on error, more extensive logs

### PR DESCRIPTION
**Pull request by ah1053**
1. Don't Stop EdgeEvents when an error occurs.
2. More extensive logs*.

*(JSONUtility.toJSON needs Serializable attribute to be added to C# SDK edge events) and it doesn't support nested deserialization.

### CODE REVIEW

**Comment by brucearmstrong** [comment]
I don't see a change involving stopping Edge Events. More to come?

File: Runtime/Scripts/EdgeEventsManager.cs [hunk 1]
```diff
@@ -553,7 +573,6 @@ void PropagateError(FindCloudletEventTrigger trigger, string error_msg)
       findCloudletEvent.trigger = trigger;
       Debug.LogError(error_msg);
       EdgeEventsStatus eventStatus = new EdgeEventsStatus(Status.error, error_msg);
-      StopEdgeEvents();
```
Comment (ah1053) [inline Runtime/Scripts/EdgeEventsManager.cs:556]: Only this change here involving EdgeEvents logic @brucearmstrong 
Basically don't stop edge events if there is an error

**Review by lgarner — APPROVED** [review]
Ok. Just simplify the ToString() handling, or a separate ticket to fix the verbosity with an override.

**Review by brucearmstrong — APPROVED** [review]

**Comment by ah1053** [comment]
@lgarner  created this for the toString override for EdgeEventsConfig and UpdateConfig https://mobiledgex.atlassian.net/browse/EDGECLOUD-5606

**Pull request merged**